test/integration/singlecluster/webhook/core/clusterqueue_test.go

```diff
@@ -583,7 +583,68 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", ginkgo.Ordered, func() {
 					},
 				},
 				gomega.Succeed()),
-			ginkgo.Entry("Should forbid negative weight",
+			ginkgo.Entry("Should allow small FairSharing weight",
+				&kueue.ClusterQueue{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "cluster-queue",
+					},
+					Spec: kueue.ClusterQueueSpec{
+						FairSharing: &kueue.FairSharing{
+							Weight: ptr.To(resource.MustParse("1m")),
```

Comment:
nit: lets add comments to what these units are, similar to what you did in the webhook test
so // 10^-3

Same for ones below

Reply:
Done.